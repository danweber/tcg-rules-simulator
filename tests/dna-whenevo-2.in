VERSION:2
# plain evo and don't get the [if dna] effect. TODO: recognize "when"
FIELD1:Blue,CS2-Colossal,CS2-01 Yellow,CS1-Obsidian,CS1-01
HAND1:CS2-Rampager
FIELD2:5k,DW1-Van One,CS2-03 Two,CS2-03 Three,CS2-02
GAME:P1 T4 MAIN 7
DUMP:DP

ECHO: --- FUZOR ---


#  card, left, cost, [right, newlabel (new label not used yet)]
CMD:EVO CS2-Rampage Blue
STEP:AUTOANSWER:CAPTURE
DUMP:DP
