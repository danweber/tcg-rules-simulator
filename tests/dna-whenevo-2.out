FIELD1:Blue,CS2-12,CS2-01,7500DP Yellow,CS1-10,CS1-01,13000DP
FIELD2:5k,DW1-Vanillamon,5000DP One,CS2-03,1500DP Two,CS2-03,1500DP Three,CS2-02,2000DP
GAME:P1 T4 MAIN 7
ECHO: --- FUZOR ---
CAPTURE:Target VanillaMon 3
CAPTURE:Target Long Sword 4
CAPTURE:Target Long Sword 5
CAPTURE:Target Sharp Blade 6
FIELD1:Blue,CS2-Rampager,CS2-12,CS2-01,14000DP Yellow,CS1-10,CS1-01,13000DP
FIELD2:One,CS2-03,1500DP Two,CS2-03,1500DP Three,CS2-02,2000DP
GAME:P1 T4 MAIN 2
